platforms/common/src/main/resources/assets/dynamic_fps/lang/de_at.json

```diff
@@ -15,11 +21,41 @@
     "config.dynamic_fps.idle_time": "Untätigkeits-Timeout",
     "config.dynamic_fps.idle_time_tooltip": "Zeit in Minuten ohne Aktivität, bis die Untätig Konfiguration aktiviert wird, während das Spiel fokussiert ist.",
 
+    "config.dynamic_fps.idle_condition": "Untätigkeits-Bedingung",
+    "config.dynamic_fps.idle_condition_none": "Keine",
+    "config.dynamic_fps.idle_condition_on_battery": "Im Akkubetrieb",
+
+    "config.dynamic_fps.battery_tracker": "Akku-Tracker",
+    "config.dynamic_fps.battery_tracker_tooltip": "Schalte alle Akku-Features ein oder aus.",
+
+    "config.dynamic_fps.battery_tracker_switch_states": "Nutze den Auf Akku Status",
```

Comment:
It's technically incorrect to combine words with spaces in German, and I think it reads weirdly here. Maybe `"Auf Akku"-Status` or `Auf-Akku-Status` would be better?